api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/allowExportSelectedData.md

```diff
@@ -5,13 +5,15 @@ default: false
 ---
 ---
 ##### shortDescription
-Allows an end user to export selected rows only.
+Allows users to export selected rows only.
 
 ---
 When this option is set to **true**, a click on <img src="/Content/images/doc/20_1/DataGrid/icons/toolbar_export.png" alt="DevExtreme DataGrid HTML5 Toolbar Exporting" style="vertical-align:middle"/> invokes a menu that contains the "Export selected rows" command.
 
 ![DevExtreme DataGrid Exporting Selected Rows](/images/DataGrid/Export_SelectedRows.png)
 
+[note]This option is disabled when summary calculation is [remote](Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/remoteOperations/#summary) and [custom summary calculation](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/summary/#calculateCustomSummary) is used.
```

Comment:
"option is disabled" - do we use "disabled" when describing properties of our widget?
I think it's about end-user UI and it's better to write "option is not supported".

Reply:
And, let's specify the source issue for any change: since 2021.01.01, there will be "WTF?" only about this text.
This scenario we investigated in t852278 ticket (it's ok to refer to private tickets in subjects in devextreme repo).

Reply:
Sorry, but I cannot agree that 'disabled' does not fit here. Let me show a few examples:
1. [The showCancelButton option makes sense only if the usePopover option is disabled.](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxActionSheet/Configuration/#showCancelButton)
2. [If paging is disabled, the widget loads all rows simultaneously which can reduce the widget's performance.](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxDataGrid/Configuration/scrolling/#mode)
3. [If this option is disabled, the widget does not display values until the number of typed characters exceeds the minSearchLength value, otherwise it displays all values regardless of the specified filtering condition.](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxLookup/Configuration/#showDataBeforeSearch)

And there are many more, so I would like to keep it.

Considering the tickets - looks ok to me. If @RomanTsukanov has no objections to this, we might extend this practice.

Reply:
Is it really important to refer to the article on [custom aggregate funtion](https://js.devexpress.com/Documentation/Guide/Widgets/DataGrid/Summaries/Custom_Aggregate_Function/)? It seems to me that it is not related to this description